data/human/weapons.txt

```diff
@@ -1300,6 +1365,65 @@ effect "nuke residue slow"
 	"velocity scale" 0.1
 
 
+outfit "Barrage Missile"
+	category "Ammunition"
+	cost 200
+	thumbnail "outfit/barrage sprite"
+	"mass" .1
+	"barrage capacity" -1
+	description "A small missile designed by the Republic Navy to Give fighters and drones more firepower, it was purpose built to be fired in large waves, hence its name."
+
+outfit "Barrage Storage"
+	category "Ammunition"
+	cost 10000
+	thumbnail "outfit/barrage storage"
+	"mass" 3
+	"outfit space" -5
+	"barrage capacity" 200
+	ammo "Barrage Missile"
+	description "A safe Storage for the Barrage Missiles."
+
+outfit "Barrage Launcher"
+	category "Secondary Weapons"
+	cost 5000
+	thumbnail "outfit/barrage launcher"
+	"mass" 3
+	"outfit space" -3
+	"weapon capacity" -3
```

Comment:
Outside of Zitchas's new fighter pods, the smallest missile launcher is 10 tons. I get that this is supposed to be compact, but only 3 tons seems excessively small. Could we make it a bit larger? Totally open to suggestions on a new number though.

Reply:
I think that might be about the same size as the javelin mini pods... (The fighter sized one that only holds 40 javelins)

Reply:
Ah, gotcha. In that case I would suggest adding "Pod" to the name for the sake of clarity, but it's not a huge thing. Feel free to resolve

Reply:
Well it's made to be pod sized, so I don't think pod is required in the name, especially since there is no larger variants, at least not currently